go.mod

```diff
@@ -106,8 +106,8 @@ require (
 	github.com/zeebo/incenc v0.0.0-20180505221441-0d92902eec54 // indirect
 	go.uber.org/atomic v1.3.2 // indirect
 	go.uber.org/multierr v1.1.0 // indirect
-	go.uber.org/zap v1.9.1
-	golang.org/x/crypto v0.0.0-20180910181607-0e37d006457b
+	go.uber.org/zap v1.8.0
```

Comment:
revert change to zap version

Reply:
very puzzling, I rebased my branch to master with CTRL+C changes and go.mod file shows the zap version as v1.8.0.  I will keep an eye on it when I am merging.

Reply:
This still needs updating

Reply:
changed to v1.9.1